### PR TITLE
Add a new OBP device

### PR DESCRIPTION
**Pull request by norbert-walter**
#### Summary
Add the company Open Boat Projects and a LoRa device for a boat monitor

#### Changes
No changes, it is a initial request

#### Notes for Reviewers
That is my listed first device for TTN V3

#### Release Notes
- Initial released

### CODE REVIEW

**Review by Jaime-Trinidad — APPROVED** [review]

File: vendor/index.yaml [hunk 1]
```diff
@@ -1969,3 +1969,9 @@ vendors:
     logo: koidra-logo.svg
     social:
       linkedin: https://www.linkedin.com/company/koidra/
+      
+  - id: open-boat-projects
+    name: Open Boat Projects
+    description: Open Boat Projects is a platform on which water sports enthusiasts present their self-constructed DIY projects around boating. The focus is on the idea of open source and open hardware as well as the exchange of ideas.
+    website: https://open-boat-projects.org
+    logo: obp_logo.jpg
```
Comment (Jaime-Trinidad) [inline vendor/index.yaml:1977]: change logo extension to .png, in index .jpg is not valid

File: vendor/open-boat-projects/lora_boat_monitor_1.jpg
Comment (Jaime-Trinidad) [inline vendor/open-boat-projects/lora_boat_monitor_1.jpg:1]: resize your image maximum is 2000x2000

File: vendor/open-boat-projects/lora_boat_monitor_2.jpg
Comment (Jaime-Trinidad) [inline vendor/open-boat-projects/lora_boat_monitor_2.jpg:1]: resize your image maximum is 2000x2000

**Comment by Jaime-Trinidad** [comment]
Hi @norbert-walter please review the comments to fix some details, then run again `make validate` and `make validate fmt` thanks

File: vendor/open-boat-projects/loraboatmonitor-profile.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+#vendorProfileID: 0
+
+# LoRaWAN MAC version: 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4 or 1.1
+macVersion: 1.0
```
[the remark this answers is not in the document]
Comment (norbert-walter) [reply vendor/open-boat-projects/loraboatmonitor-profile.yaml:7]: Is a old hardware modul. I think 1.0.1 is also working.

**Comment by norbert-walter** [comment]
Thanks for your visit. I changed the files.

File: vendor/open-boat-projects/loraboatmonitor-profile.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jaime-Trinidad) [reply vendor/open-boat-projects/loraboatmonitor-profile.yaml:7]: what I mean 1.0 should be inside single quotation mark  i.e. '1.0'

File: vendor/index.yaml (hunk 1 above)
Comment (Jaime-Trinidad) [reply vendor/index.yaml:1977]: @norbert-walter don't forget to change the extension to your logo and here to .png

**Comment by norbert-walter** [comment]
Thanks for your help.

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**